Form/DoctrineMongoDBTypeGuesser.php

```diff
@@ -49,6 +49,10 @@ public function guessType($class, $property)
 
         list($metadata, $name) = $ret;
 
+        if (! $metadata->hasField($property)) {
+            return new TypeGuess($this->typeFQCN ? TextType::class : 'text', [], Guess::LOW_CONFIDENCE);
```

Comment:
Now I saw the test I'm wondering whether guessing `TextType` (even with low confidence) is correct. From what I saw in https://symfony.com/doc/current/form/type_guesser.html it's possible to just `return;` if we can't (or shouldn't) make a guess at all. Do you think it'd be more appropiate?

Reply:
I copy behaviour from a [symfony/doctrine-bridge](https://github.com/symfony/symfony/blob/master/src/Symfony/Bridge/Doctrine/Form/DoctrineOrmTypeGuesser.php#L88). Looks like in the first versions type `guessType` returns only `TypeGuess` instance (based on phpDoc). I think now we can return nulls

Reply:
Also if you could squash commits later that'd be awesome :)

Reply:
Done